### PR TITLE
[doc] SonataTranslationBundle + SonataAdminBundle

### PR DESCRIPTION
**Pull request by rmed19**
Add stof stof_doctrine_extensions configuration to fix saving problem with SonataAdminBundle

issue #21

### CODE REVIEW

File: Resources/doc/reference/orm.rst [hunk 1]
```diff
@@ -16,8 +16,8 @@ First step, your entities have to implement the `TranslatableInterface`_.
 Todo do so ``SonataTranslationBundle`` brings some base classes you can extend.
 Depends on how you want to save translations you can choose between :
 
-* `Sonata\TranslationBundle\Model\Gedmo\AbstractTranslatable`
-* `Sonata\TranslationBundle\Model\Gedmo\AbstractPersonalTranslatable`
+* `Sonata\\TranslationBundle\\Model\\Gedmo\\AbstractTranslatable`
+* `Sonata\\TranslationBundle\\Model\\Gedmo\\AbstractPersonalTranslatable`
```
Comment (OskarStark) [inline Resources/doc/reference/orm.rst]: why?

Comment (rmed19) [inline Resources/doc/reference/orm.rst]: To escape backslash. It is a special character

Comment (OskarStark) [inline Resources/doc/reference/orm.rst]: i would wrap it in a code block

File: Resources/doc/reference/installation.rst [hunk 2]
```diff
@@ -21,6 +21,7 @@ Now, enable the bundle in the kernel:
         return array(
             // ...
             new Sonata\TranslationBundle\SonataTranslationBundle(),
+            new Stof\DoctrineExtensionsBundle\StofDoctrineExtensionsBundle(),
```
Comment (nicolas-bastien) [inline Resources/doc/reference/installation.rst]: Hi @rmed19 StofDoctrineExtensionsBundle is not required with this bundle so I would like you to move this part in Resources/doc/reference/orm.rst.

As you can see there is a section called : 3.2. DEFINE TRANSLATABLE FIELDS which point to gedmo documentation, you can add a part : 

If you work with StofDoctrineExtensionsBundle, be sure to activate translatable extension in your configuration

(when you copy/paste your configuration example please put 4 spaces to indent and not 8).

After that I'll merged

**Comment by core23** [comment]
ping @rmed19

**Comment by dbu** [comment]
ping @rmed19

**Comment by core23** [comment]
Please squash and rebase @rmed19

File: Resources/doc/reference/installation.rst [hunk 3]
```diff
@@ -44,6 +45,11 @@ To use the ``TranslationBundle``, add the following lines to your application co
                 enabled: true
             #phpcr:
             #    enabled: true
+        stof_doctrine_extensions:
+                default_locale: %locale%
```
Comment (OskarStark) [inline Resources/doc/reference/installation.rst]: indention is wrong

**Comment by OskarStark** [comment]
No feedback from @rmed19 

@rmed19 feel free to reopen this PR if you would like to invest some time applying the comments 👍

**Pull request closed without merging**